### PR TITLE
Fix cart modal when a customized product is added to cart

### PR DESCRIPTION
**Pull request by unlocomqx**
When a customized product is added, the modal dialog shows the quantity of the non customized product quantity
Here's how to reproduce:
- Add a non customized product to cart (quantity = 2 for example)
- Add the same product but after saving a customization and setting the qty to 3
- The modal shows up and displays quantity: 2

### CODE REVIEW

**Comment by unlocomqx** [comment]
Please advise, this is causing some issues/confusion

**Comment by unlocomqx** [comment]
Please review this PR, this bug is still unsolved

**Comment by xpech** [comment]
More interesting : if you set a specific price to the customization, the modal show the basic product price and not the real shopping cart item (product price + customization price). The messaging system of Prestashop (core.js / emit('updateCart')) did not sent the real shopping cart item so the module cannot show the real shopping datas. bad Idea !!!

**Comment by unlocomqx** [comment]
@xpech Hi, by applying my fix, I think that the correct combined price is displayed
But anyway, I will test that when this gets merged hopefully

**Comment by unlocomqx** [comment]
I will make a PR to core to add the `id_customization` field instead of reading it from the DOM. Then I will update this PR, cheers!

**Comment by unlocomqx** [comment]
I made this new PR to add the customization ID to the event data
https://github.com/PrestaShop/PrestaShop/pull/8956

Cheers

**Comment by marionf** [comment]
It's ok for functional tests

File: ps_shoppingcart.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ $(document).ready(function () {
 
         if (event && event.reason) {
           requestData = {
+            id_customization: $('#product_customization_id').val(),
```
Comment (Quetzacoalt91) [inline ps_shoppingcart.js]: It seems we still need to modify that line before validating the PR, doesn't it?

**Comment by unlocomqx** [comment]
I apologize but I'd like to bump this up :smile:

**Comment by matks** [comment]
> I apologize but I'd like to bump this up 😄

No need to apologize 😉 sometimes it's a good thing to remind us about it as there are so many things to do we can forget some, thanks !

**Comment by unlocomqx** [comment]
@matks Yeah, I just saw stats for 1.7.6 beta, the number of PRs is huge!
Regarding this PR, it's well tested with a number of customers who encountered the issue when working with customizations, I wish it would make it to 1.7.6
Cheers

**Comment by matks** [comment]
@marionf I know it has been validated in the past but it was so long ago that I would like us to re-check it if possible 🙏

**Comment by marionf** [comment]
It's all good with 1.7.6 :+1:

**Comment by matks** [comment]
Thank you @unlocomqx for the PR and thank you @xpech for the review 😉

**Comment by unlocomqx** [comment]
Glad to contribute, Thank you!

**Pull request merged**